app/views/scrapers/new.html.haml

```diff
@@ -6,15 +6,20 @@
       .col-sm-6
         = f.input :original_language_key, collection: Morph::Language.languages_supported.map{|l| [l.human, l.key, {"data-content" => "#{image_tag(l.image_path, size: '20x20')} #{l.human}"}]}, include_blank: false, hint: "The language you would like to write your scraper in. You can change this later.", label: "Language"
     .row
-      .col-sm-2
-        = f.input :owner_id, collection: current_user.all_owners.map{|o| [o.nickname, o.id, {"data-content" => "#{owner_image(o, 20, false)} #{o.nickname}"}]}, include_blank: false
-      .col-sm-6
-        -# Using long-winded form here because simple_form doesn't directly support prepends
-        = f.input :name do
-          .input-group
-            %span.input-group-addon /
-            = f.input_field :name, class: "form-control", placeholder:"city_of_sydney_development_applications"
-          %span.help-block Name to give the scraper here and on GitHub
+      - if current_user.all_owners.count == 1
+        = f.input :owner_id, as: "hidden", input_html: { value: current_user.id }
+        .col-sm-6
+          = f.input :name, placeholder: "city_of_sydney_development_applications", hint: "Name to give scraper here and on Github"
```

Comment:
There's now duplication of the placeholder and hint text between those two variants of it. Since simple_form doesn't support prepends one of the variants isn't using simple_form. If you write this variant in its long form not using simple_form, you'll probably find that you can simplify this a lot and at the very least remove the duplication of placeholder and hint.

Whenever you see that kind of duplication (and even more so when you create it!) you should work hard to get rid of it.